tokio/tests/task_id.rs

```diff
@@ -0,0 +1,287 @@
+#![warn(rust_2018_idioms)]
+#![allow(clippy::declare_interior_mutable_const)]
+#![cfg(all(feature = "full", tokio_unstable, not(tokio_wasi)))]
```

Comment:
Except for the panic-location tests, the tests here should work under wasm. You can disable specific tests under wasm with `#[cfg_attr(tokio_wasm, ignore)]`.
```suggestion
#![cfg(all(feature = "full", tokio_unstable))]
```